docs/getting-started/overview.md

```diff
@@ -35,29 +35,30 @@ Zowe consists of the following components:
 
 ### Zowe Launcher
 
-Provides an advanced launcher for Zowe z/OS server components in a high availability configuration. It performs the following operations:
+The Zowe Launcher makes it possible to launch Zowe z/OS server components in a high availability configuration, and performs the following operations:
 
-- Start all Zowe server components using the 'START' (or 'S') operator command.
-- Stopping the Zowe server components using the `STOP` (or `P`) operator command
-- Stopping and starting specific server components without restarting the entire Zowe instance using `MODIFY` (or `F`) operator command
+- Start all Zowe server components using the `START` (or `S`) operator command.
+- Stop Zowe server components using the `STOP` (or `P`) operator command.
+- Stop and start specific server components without restarting the entire Zowe instance using `MODIFY` (or `F`) operator command.
 
 ### API Mediation Layer
 
-The API Mediation Layer provides a single point of access for mainframe services APIs. It also provides a [Single Sign On (SSO)](../extend/extend-apiml/api-mediation-sso.md) capability for mainframe users.
+The API Mediation Layer provides a single point of access for APIs of mainframe services, and provides a [Single Sign On (SSO)](../extend/extend-apiml/api-mediation-sso.md) capability for mainframe users.
 
-<details>
-<summary> Learn more </summary>
-The API Mediation Layer facilitates secure communication between loosely coupled clients and services through variety of API types, such as REST, GraphQL or Web-Socket. 
-It consists of three core components: the API Gateway, the Discovery Service, and the API Catalog:
+The API Mediation Layer (API ML) facilitates secure communication between loosely coupled clients and services through a variety of API types, such as REST, GraphQL or Web-Socket. 
+API ML consists of three core components: the API Gateway, the Discovery Service, and the API Catalog:
 
- The API Gateway provides secure routing of API requests from clients to registered API services.
- The Discovery Service allows dynamic registration of microservices and enables their discoverability and status updates.
- The services Catalog provides a user-friendly interface to view and try out all registered services, read their associated APIs documentation in OpenAPI/Swagger format.
+- The API Gateway provides secure routing of API requests from clients to registered API services.
+- The Discovery Service allows dynamic registration of microservices and enables their discoverability and status updates.
+- The API Catalog provides a user-friendly interface to view and try out all registered services, read their associated APIs documentation in OpenAPI/Swagger format.
 
 Core Zowe also provides out of the box services for working with MVS Data Sets, JES, as well as working with z/OSMF REST APIs.
```

Comment:
We should mention that these are deprecated and will be removed in V3

Reply:
Please specify what will be deprecated in v3.

Reply:
Done